heron/tracker/src/python/handlers/metricshandler.py

```diff
@@ -29,6 +29,7 @@ class MetricsHandler(BaseHandler):
   URL - /topologies/metrics
   Parameters:
    - cluster (required)
+   - role - Name of person who submits the topology (optional)
```

Comment:
plese follow the comment format here.

`- parameter_name (required) explainations`